src/app/ReadHandler.cpp

```diff
@@ -115,6 +116,7 @@ void ReadHandler::Shutdown(ShutdownOptions aOptions)
     mIsChunkedReport           = false;
     mInitiatorNodeId           = kUndefinedNodeId;
     mHoldSync                  = false;
+    mInitialWrittenEventsBytes = 0;
```

Comment:
Rename to `mLastWrittenEventsBytes` for better understanding.

Reply:
Done, thanks